CMakeLists.txt

```diff
@@ -131,7 +131,9 @@ else (APPLE)
 endif(APPLE)
 
 set(CMAKE_INCLUDE_CURRENT_DIR TRUE)
-set(CMAKE_BUILD_WITH_INSTALL_RPATH ON)
+if (NOT CMAKE_SKIP_RPATH) # Option to not set rpath - required for packaging on Debian
```

Comment:
I would prefer without the IF test but it's not that important. If you have some time to try, please do, I'm curious if it will work as documented.